### ISSUE TITLE
How to use "armnnCaffeParser + Squeezenet" correctly?

### ISSUE THREAD

**Issue by Lost-sledgehammer** [ARM-software/armnn#125]
hi all

I did the following test:

./ExecuteNetwork18.08 -f caffe-binary -m ./model/bs=1_converted_squenetv1.0_ft_20190107__iter_5000.caffemodel -c CpuRef -i data -d ./image_0001.jpg -o prob

Get the following error:
ArmNN v20180800
Armnn Error: kernel_h=15 and kernel_w=15 must be equal and within the valid rangeof [0, 11]  at function ParsePoolingLayer [/mnt/LVM_40G/gcc5.4-2017.01/armnn/src/armnnCaffeParser/CaffeParser.cpp:950]

ps:
1.	I modified the ExecuteNetwork.cpp using opencv to decode the JPG photo.
2.	squeezenet_v1.0.caffemodel without finetune gets the same error as above

meanwhile, I see the issue (https://github.com/ARM-software/armnn/issues/65), so I have a few questions:

1. Does armnnCaffeParser support Squeezenet properly? 
2. How did you test it?  Can you show me more details about the operation? (sry，I am a novice.☺)
3. What's the progress on this issue(https://github.com/ARM-software/armnn/issues/65)?

**Comment by Lost-sledgehammer** [comment]
Many thanks

**Comment by MatthewARM** [comment]
Hi @Lost-sledgehammer we think 4D Softmax should now be working in the master branch (on Gerrit at https://review.mlplatform.org/#/admin/projects/ml/armnn). You would need an updated Compute Library as well.

I'll see if I can get someone to answer your other questions - I haven't personally looked at Squeezenet 1.0.

**Comment by Lost-sledgehammer** [comment]
@MatthewARM 
  hi, MatthewARM. 
  Thank you for your reply.
  I will make some attempts based on your reply.
  BTW,I would love to know the answer to my question1 (or any successful experience about it), and if the answer is positive, it will give me great confidence.
   Many thanks

**Comment by MatthewARM** [comment]
Hi @Lost-sledgehammer I hope you were able to get Squeezenet working? The 19.02 release is out now so hopefully this solves the issue?

**Comment by MatthewARM** [comment]
Closing due to inactivity, please get back in touch if you are still having problems.

All the best,
Matthew
